pkg/op-data/op_data/sources/discourse/topics.py

```diff
@@ -111,5 +111,14 @@ async def acquire_and_save():
 
     @staticmethod
     async def update_relationships():
-        # If there are any relationships to update, implement them here
-        pass
+        topics = await Topic.all().prefetch_related('rawTopic')
+
+        for topic in topics:
+            raw_related_topics = topic.rawTopic.rawData.get("related_topics", None)
+
+            if isinstance(raw_related_topics, list):
+                related_topics_ids = [item['id'] for item in raw_related_topics]
+
+                related_topics = await Topic.filter(externalId__in=related_topics_ids)
+
+                await topic.relatedTopics.add(*related_topics)
```

Comment:
`#could` bulk add them instead of doing one by one as this will take more time. Also what happens if the related topics are the same from the ones that already exist?